e2e/cypress/integration/account_settings/display/message_display_mode_spec.js

```diff
@@ -12,55 +12,53 @@
 
 import {getRandomId} from '../../../utils';
 
-describe('Account Settings > Display > Message Display', () => {
+describe('Messaging', () => {
     before(() => {
         // # Login as new user and visit town-square
         cy.apiInitSetup({loginAfter: true}).then(({team}) => {
             cy.visit(`/${team.name}/channels/town-square`);
         });
     });
 
-    ['COMPACT', 'STANDARD'].forEach((display) => {
-        it(`M14283 ${display} view: Line breaks remain intact after editing`, () => {
-            cy.uiChangeMessageDisplaySetting(display);
+    it('MM-T103 Compact view: Line breaks remain intact after editing', () => {
+        cy.uiChangeMessageDisplaySetting('COMPACT');
```

Comment:
I wonder if we should still test Standard view as well. Could be on another step or a separate test case. Thoughts?

Reply:
That's a good point. I've added a second test step for the standard view instead of creating a whole new test case. I think that simplifies it a lot.
Also, I've moved it from "Messaging" to "Account Settings" in Zephyr to address Joseph's concern.
Re-requesting review -- thanks @saturninoabril!